docs/usage.md

```diff
@@ -23,12 +23,19 @@ A relatively trivial example of an analysis can be found in this repo [here](../
 
 Enclave containers can access the following data CSV files:
 
-* [assessments.csv](#assessmentscsv)
-* [courses.csv](#coursescsv)
-* [enrollments.csv](#enrollmentscsv)
-* [grades.csv](#gradescsv)
-* [oneroster_demographics.csv](#oneroster_demographicscsv)
-* [users.csv](#userscsv)
+- [RAISE Enclave Usage](#raise-enclave-usage)
+  - [Operational model](#operational-model)
+  - [Data model](#data-model)
+  - [`assessments.csv`](#assessmentscsv)
```

Comment:
I find that using the backticks on the filenames as you've modified is kind of an eye strain in the rendered HTML:

![image](https://user-images.githubusercontent.com/839025/213545148-24c7ca56-c379-43cc-ab51-7ea7c48b8b8c.png)

vs 

![image](https://user-images.githubusercontent.com/839025/213545183-424781dc-b3ef-4f4a-a08b-34150af15374.png)

Do you prefer the latter? Just wondering if this was intentional :man_shrugging: .

Reply:
Yea none of this is intentional - these edits were produced by auto-format on save in vscode that I'm having trouble finding the setting for 🙄 infuriating.